style.css

```diff
@@ -48,3 +49,9 @@ header {
 .links {
   text-align: center;
```

Comment:
This seems an odd name, reading "left" and all the class is doing is center something. ;-)

Reply:
![germans](https://user-images.githubusercontent.com/54703643/110030011-7a170600-7d35-11eb-82c2-5748fa083557.jpg)